lib/ferrum/page/tracing.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module Ferrum
+  class Page
+    class Tracing
+      INCLUDED_CATEGORIES = %w[
+        devtools.timeline
+        v8.execute
+        disabled-by-default-devtools.timeline
+        disabled-by-default-devtools.timeline.frame
+        toplevel
+        blink.console
+        blink.user_timing
+        latencyInfo
+        disabled-by-default-devtools.timeline.stack
+        disabled-by-default-v8.cpu_profiler
+        disabled-by-default-v8.cpu_profiler.hires
+      ].freeze
+      EXCLUDED_CATEGORIES = %w[
+        *
+      ].freeze
+
+      def initialize(client:)
+        self.client = client
+      end
+
+      def start(trace_params: {}, **options)
+        self.options = {
```

Comment:
It's mixed up right with optiosn and params and it's hard to judge what goes where. Need to think about it

Reply:
yup, see this thread: https://github.com/rubycdp/ferrum/pull/189#discussion_r766151731
let's drop `trace_params` for now